include/Track.h

```diff
@@ -294,6 +294,11 @@ class TrackContentObjectView : public selectableObject, public ModelView
 	void setBBPatternBackground( const QColor & c );
 	void setGradient( const bool & b );
 	void setMouseHotspotHand(const QSize & s);
+	
+	QColor color() const
+	{
+		return( m_tco->colorRgb() );
```

Comment:
Should this be:
```suggestion
		return( m_tco->color() );
```
? Based on the name and return type.

Reply:
Ugg... I probably didn't notice this from when I was renaming stuff... thanks for pointing it out, I'll update it

Reply:
Just to note, `return` doesn't need parenthesis around it.